python/tests/api/writer/test_whylabs_integration.py

```diff
@@ -0,0 +1,100 @@
+import os
+import time
+from uuid import uuid4
+
+import pytest
+from whylabs_client.api.dataset_profile_api import DatasetProfileApi
+from whylabs_client.model.profile_traces_response import ProfileTracesResponse
+
+import whylogs as why
+from whylogs.api.writer.whylabs import WhyLabsWriter
+from whylogs.core import DatasetProfileView
+from whylogs.core.feature_weights import FeatureWeights
+from whylogs.core.schema import DatasetSchema
+
+# TODO: These won't work well if multiple tests run concurrently
+
+
+ORG_ID = "org-0"
+MODEL_ID = "model-2250"
+
+os.environ["WHYLOGS_NO_ANALYTICS"] = "True"
+os.environ["WHYLABS_API_ENDPOINT"] = "https://songbird.development.whylabsdev.com"
+os.environ["WHYLABS_DEFAULT_ORG_ID"] = ORG_ID
+os.environ["WHYLABS_DEFAULT_DATASET_ID"] = MODEL_ID
+# WHYLABS_API_KEY needs to come from the environment
+
+
+@pytest.mark.load
```

Comment:
Perhaps `s/load/integration/`?

Reply:
We could add a new pytest mark value for "integration". But let's do some of that in a follow up. Better to get the initial test capabilities and fixes in first.